lib/providers/services/eos_service.dart

```diff
@@ -3,6 +3,10 @@ import 'package:flutter/widgets.dart' show BuildContext;
 import 'package:provider/provider.dart';
 import 'package:seeds/constants/config.dart';
 import 'package:seeds/constants/http_mock_response.dart';
+import 'package:dart_esr/dart_esr.dart' as ESR;
+
+String chainId =
```

Comment:
How is this created? 
Why is this needed?
Why use this over default? 


/** Chain to use, defaults to EOS main-net if omitted. */
  String chainId;



@n13

Reply:
There are multiple blockchains based on eosio software, and each of them has unique chainId to differentiate between them. 
We are building on Telos blockchain, and here we specify it's chainId, - so transactions will be signed uniquely for Telos blockchain.